homeassistant/components/sensor/zwave.py

```diff
@@ -7,15 +7,27 @@
 at https://home-assistant.io/components/zwave.html
 """
 # pylint: disable=import-error
+from homeassistant.helpers.event import track_point_in_time
 from openzwave.network import ZWaveNetwork
 from pydispatch import dispatcher
-
+import datetime
+import homeassistant.util.dt as dt_util
 import homeassistant.components.zwave as zwave
 from homeassistant.helpers.entity import Entity
 from homeassistant.const import (
     ATTR_BATTERY_LEVEL, STATE_ON, STATE_OFF,
     TEMP_CELCIUS, TEMP_FAHRENHEIT, ATTR_LOCATION)
 
+PHILIO = '013c'
+PHILIO_SLIM_SENSOR = '0002'
+PHILIO_SLIM_SENSOR_MOTION = (PHILIO, PHILIO_SLIM_SENSOR, 0)
+
+WORKAROUND_NO_TRIGGER_OFF_EVENT = 'trigger_no_off_event'
+
+SPECIFIC_DEVICE_MAPPINGS = [
```

Comment:
If we structure this value differently, we should be able to do instant lookups on it.

```python
SPECIFIC_DEVICE_MAPPINGS = {
  (zwave.COMMAND_CLASS_SENSOR_BINARY, '013c', '0002'): WORKAROUND_NO_TRIGGER_OFF_EVENT,
}
```